rpc/runtime-api/src/identity.rs

```diff
@@ -17,9 +15,6 @@ sp_api::decl_runtime_apis! {
         /// Returns CDD status of an identity
         fn is_identity_has_valid_cdd(did: IdentityId, buffer_time: Option<u64>) -> CddStatus;
 
-        /// Returns DID of an asset
-        fn get_asset_did(ticker: Ticker) -> AssetDidResult;
```

Comment:
Should bump the API version number, since clients can use it to check if this call is available.